[Unity 2019] No overload for method 'DoAddButton' takes 2 arguments

**Describe the bug**
Installing tri inspector results in this error in the console:
> Library\PackageCache\com.codewriter.triinspector@0e23ca4a42\Unity.InternalAPIEditorBridge.012\ReorderableListProxy.cs(66,31): error CS1501: No overload for method 'DoAddButton' takes 2 arguments

**Unity version:** 2019.4.36f1
**Tri Inspector version:** 0e23ca4a42 (1.13.2)

--- a/Editor/Editors/TriScriptedImporterEditor.cs
+++ b/Editor/Editors/TriScriptedImporterEditor.cs
@@ -1,7 +1,12 @@
 ﻿using TriInspectorUnityInternalBridge;
 using UnityEditor;
-using UnityEditor.AssetImporters;
 using UnityEngine.UIElements;
+
+#if UNITY_2020_2_OR_NEWER
+using UnityEditor.AssetImporters;
+#else
+using UnityEditor.Experimental.AssetImporters;
+#endif
 
 namespace TriInspector.Editors
 {

--- a/Editor/Elements/TriBoxGroupElement.cs
+++ b/Editor/Elements/TriBoxGroupElement.cs
@@ -18,7 +18,7 @@ namespace TriInspector.Elements
         private bool _expanded;
 
         [Serializable]
-        public struct Props
+        public new struct Props
         {
             public string title;
             public TitleMode titleMode;

--- a/Unity.InternalAPIEditorBridge.012/ReorderableListProxy.cs
+++ b/Unity.InternalAPIEditorBridge.012/ReorderableListProxy.cs
@@ -63,7 +63,11 @@ namespace TriInspectorUnityInternalBridge
 
         public static void DoAddButton(ReorderableList list, Object value)
         {
+#if UNITY_2020_2_OR_NEWER
             defaultBehaviours.DoAddButton(list, value);
+#else
+            defaultBehaviours.DoAddButton(list);
+#endif
         }
     }
 }

